### PR TITLE
added a tooltip [skip ci pytest]

### PR DESCRIPTION
**Pull request by asf-algoo**
## Checkpoints

Added a tooltip to the attach files icon, with translations

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import { translate } from 'react-i18next'
 import PopupUploadFile from '../../container/PopupUploadFile.jsx'
 import IconButton from '../Button/IconButton.jsx'
+import { t } from 'i18next/dist/commonjs'
```
Comment (raphj) [inline frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx:6]:
```suggestion
```

Your IDE probably added this line automatically when your were typing

**Review by raphj — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx (hunk 1 above)
Comment (Skylsmoi) [inline frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx:6]: this import doesn't seem to be used
and I don't think you're supposed to import anything from i18next, you should only import from 'react-i18next'

**Review by Skylsmoi — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx (hunk 1 above)
Comment (asf-algoo) [reply frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx:6]: yes it did, i'll remove it

Comment (asf-algoo) [reply frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx:6]: thanks, it was added automatically, i'll remove it

**Review by raphj — APPROVED** [review]

**Pull request merged**